deploy/terraform/main.tf

```diff
@@ -52,27 +52,27 @@ resource "null_resource" "tink_directory" {
   }
 
   provisioner "file" {
-    source      = "../../setup.sh"
+    source      = "${path.module}/../../setup.sh"
```

Comment:
I am really suprised that this worked through the `../../`,  but it did:

https://github.com/displague/terraform-metal-tinkerbell/blob/main/main.tf#L2

```
root@tink-provisioner:~# ls -la /root/tink/
total 40
drwxr-xr-x 3 root root  4096 Aug 13 22:29 .
drwx------ 6 root root  4096 Aug 13 23:24 ..
-rwxr-xr-x 1 root root   771 Aug 13 22:29 current_versions.sh
drwxr-xr-x 6 root root  4096 Aug 13 22:31 deploy
-rwxr-xr-x 1 root root  2800 Aug 13 22:29 generate-env.sh
-rw-r--r-- 1 root root     6 Aug 13 22:29 .nat_interface
-rwxr-xr-x 1 root root 13594 Aug 13 22:29 setup.sh
```